plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/actions-dropdown-menu.tsx

```diff
@@ -2,34 +2,55 @@
  * External dependencies
  */
 import { DropdownMenu } from '@wordpress/components';
-import { moreVertical, external, plugins } from '@wordpress/icons';
+import { moreVertical } from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
-export default function ActionsDropdownMenu() {
+/**
+ * Internal dependencies
+ */
+import { Subscription } from '../../types';
+
+export default function ActionsDropdownMenu( props: {
+	subscription: Subscription;
+} ) {
+	const controls = [
+		{
+			title: __( 'Manage in WooCommerce.com', 'woocommerce' ),
+			icon: <></>,
+			onClick: () => {
+				window.open(
+					'https://woocommerce.com/my-account/my-subscriptions',
+					'_blank'
+				);
+			},
+		},
+		{
+			title: __( 'Manage in Plugins', 'woocommerce' ),
+			icon: <></>,
+			onClick: () => {
+				window.location.href = '/wp-admin/plugins.php';
```

Comment:
This will break on multisites that use subfolders.
Can we use `wc.wcSettings.ADMIN_URL` to ensure the url is correct?

Reply:
`wc.wcSettings.ADMIN_URL` worked but my linter didn't like it. I [used](https://github.com/woocommerce/woocommerce/pull/41066/commits/45d85c273054479a9fa3c67c7fc38949d8516146) `import { ADMIN_URL } from '~/utils/admin-settings'` which is used all around WooCommerce Admin.